extensions/examples/landsat8-sample.json

```diff
@@ -63,7 +63,7 @@
     "self": { "rel":"self", "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00.json"},
     "alternate": { "rel":"alternate", "href": "https://landsatonaws.com/L8/153/025/LC81530252014153LGN00", "type": "html"},
     "catalog": { "rel":"catalog", "href":  "http://landsat-pds.s3.amazonaws.com/L8/catalog.json"},
-    "collection": { "rel":"collection", "href": "http://landsat-pds.s3.amazonaws.com/L8/L1T-collection.json"}
+    "cx:collection": { "rel":"cx:ollection", "href": "http://landsat-pds.s3.amazonaws.com/L8/L1T-collection.json"}
```

Comment:
Typo: `cx:ollection`.

Reply:
Fixed in https://github.com/radiantearth/stac-spec/pull/122 (among other things)